pipeline-templates/job/arm-deploy.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
+  ServiceConnection: ''
+  AppRoleAssignmentsServiceConnection: ''
+  Environment: ''
+
+jobs:
+- deployment: DeployInfrastructure
+  pool:
+    # name: DAS - Continuous Deployment Agents
+    name: DAS - Continuous Deployment
```

Comment:
Does this need changing to the new agents?

Reply:
Since the AKS agents have task cache issues, to not block the build/release split any longer we're going to run on the deployment vm